packages/@okta/vuepress-site/.vuepress/nav/main.js

```diff
@@ -55,6 +55,7 @@ module.exports = [
           { title: 'Apps', link: '/docs/api/resources/apps/'},
           { title: 'Authorization Servers', link: '/docs/api/resources/authorization-servers/'},
           { title: 'Dynamic Client Registration', link: '/docs/api/resources/oauth-clients/'},
+          { title: 'Event Hooks', link: '/docs/api/resources/event-hooks/'},
```

Comment:
@bretterer, could you confirm whether this file should part of this pull request? I don't recognize this file.

Reply:
If I remember correctly, it was added via one of the PR's that was merged into this release